modules/table/src/test/java/org/apache/ignite/internal/table/KeyValueViewOperationsSimpleSchemaTest.java

```diff
@@ -330,6 +338,119 @@ public void putGetAllTypes() {
         }
     }
 
+    /**
+     * Parametrized test for correctness of empty string insertion.
+     *
+     * @param nullable nullable param for string column.
+     */
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    public void testSingleStringInsertion(boolean nullable) {
```

Comment:
Tests in this class checks API contract.
Let's move the test with empty string to RowTest class.

Reply:
fixed, also tests that test not passed without my changes.